jenkins/release.jenkinsFile

```
@@ -14,4 +14,11 @@ standardReleasePipelineWithGenericTrigger(
             mavenArtifactsPath: "$WORKSPACE/repository/",
             autoPublish: true
         )
+        publishToArtifactsProdBucket(
+            assumedRoleName: 'sql-jdbc-upload-role',
+            source: "$WORKSPACE/libs/opensearch-sql-jdbc-shadow-*.jar",
```

Considering this is similar to https://github.com/opensearch-project/sql-odbc/blob/main/jenkins/release.jenkinsFile#L12-L17 can the `*` be `${tag}`
```suggestion
            source: "$WORKSPACE/libs/opensearch-sql-jdbc-shadow-${tag}.jar",
```

---

Thanks fixed in [19bdcc1](https://github.com/opensearch-project/sql-jdbc/pull/106/commits/19bdcc16f93f5420262134d23f80c3bc47077058)